src/intro/mod.rs

```diff
@@ -0,0 +1,92 @@
+extern crate ggez;
+use ggez::*;
+use ggez::timer;
+use ggez::nalgebra;
+use ggez::graphics;
+
+use rand::Rng;
+
+pub struct State {
+    vertigo_angle: f32,
+
+    stars: graphics::Mesh,
+
+    title: graphics::Text
+}
+
+impl State {
+    pub fn new(ctx: &mut Context) -> GameResult<State> {
+        // Generate stars
+        let mut rng = rand::thread_rng();
+
+        let mut mesh_builder = graphics::MeshBuilder::new();
+
+        for _x in 1..=320 {
```

Comment:
Style: If you don't care about the variable at all, you may prefer using `_` directly. The use of `_x` would suggest to me it is somehow related to the `x` axis or something.

Reply:
👍, addressed in b9ae58985aae2ea95723908efd9617e12d69aa8a, (fix: underscore instead of variable with misleading name)